apps/metrics/views.py

```diff
@@ -0,0 +1,62 @@
+from django.views.decorators.cache import never_cache
+from django.http import HttpResponse
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework.decorators import renderer_classes
+from django.contrib.auth.models import User
+from ..accounts.models import UserProfile
+# from ..fhir.bluebutton.models import Crosswalk
+
+
+# Metric auth and response
+@never_cache
+@api_view(['GET'])
+@renderer_classes((JSONRenderer,))
+def metric_response(request, content):
+    if request.user.is_authenticated() and request.user.is_staff:
+        return Response(content)
+    else:
+        return HttpResponse('Unauthorized', status=401)
+
+
+# User records count
+def total_user_count(request, format=None):
```

Comment:
Put all of these queries together, or look into implementing [filters](http://www.django-rest-framework.org/api-guide/filtering/).

Reply:
filters could also be future work and this could just default to the `bene` count for now.